softwareComponents/configuration/include/configuration/rofiworld.hpp

```diff
@@ -363,7 +366,8 @@ class Module {
         throw std::logic_error( "Module does not have a root component" );
     }
 
-    friend class Rofibot;
+    friend class RofiWorld
+;
```

Comment:
Proč máme takto divně středník?

Reply:
Netuším, neměnil jsem (myslím) ... a neformátuju, protože by to předělalo celý soubor. Asi by to chtělo zavést nějaký ten clang-tidy nebo tak něco...